java/java.source.base/src/org/netbeans/modules/java/source/parsing/JavacParser.java

```diff
@@ -709,9 +709,14 @@ Phase moveToPhase (final Phase phase, final CompilationInfoImpl currentInfo, Lis
         } catch (Abort abort) {
             parserError = currentPhase;
         } catch (RuntimeException | Error ex) {
```

Comment:
Perhaps this should be catching AssertionError more specifically, rather than Error in general, giving other kinds of Error the old behavior. Though I'm fine with either approach.